### ISSUE TITLE
Added icecreamburglar.blog

### ISSUE THREAD

**Issue by savornicesei** [Wyam2/docs#70]
<a href="https://github.com/moolicc"><img src="https://avatars.githubusercontent.com/u/28231505?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [moolicc](https://github.com/moolicc)**
_Monday Apr 23, 2018 at 20:28 GMT_
_Originally opened as https://github.com/Wyamio/Wyam.Web/pull/70_

----

----

_**[moolicc](https://github.com/moolicc)** included the following code: https://github.com/Wyamio/Wyam.Web/pull/70/commits_

**Comment by savornicesei** [comment]
<a href="https://github.com/daveaglick"><img src="https://avatars.githubusercontent.com/u/1020407?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [daveaglick](https://github.com/daveaglick)**
_Thursday Nov 29, 2018 at 15:35 GMT_

----

Yikes, sorry it took so long to get this merged (7 months 😨). Thanks for the link though, looks great!

**Comment by savornicesei** [comment]
<a href="https://github.com/moolicc"><img src="https://avatars.githubusercontent.com/u/28231505?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [moolicc](https://github.com/moolicc)**
_Wednesday Dec 05, 2018 at 07:45 GMT_

----

haha, it's no problemo.
Honestly I need to go and revise some of the posts I made on it to reflect what I actually coded vs. what I put down there. 
So basically I've probably not even touched it more recently than 7 months ago anyway.
